test/overview.spec.js

```diff
@@ -0,0 +1,16 @@
+const expect = require("chai").expect;
+const getTotal = require("../scripts/overview").getTotal;
+describe('overview', () => {
+    it('should get total number of captures', () => {
+        var captures = {
+            '2001': {
+                'text/html': 1,
+            },
+            '2002': {
+                'text/html': 4
+            }
+        }
+        expect(getTotal(captures)).to.equal(5);
+        expect(5).to.equal(5);
```

Comment:
We don't need this line of code. This can be deleted. Thank you!!

Reply:
Thanks for pointing that out. Its fixed.